tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
Fair enough on pointers.  I was referring to the question of same object vs. equal objects, but you're right that that's pretty clear from a programming perspective.

What's the argument for tying copy and == operations together?